ui-grid/ui-grid.d.ts

```diff
@@ -601,6 +601,13 @@ declare module uiGrid {
          */
         enableFiltering?: boolean;
         /**
+        * False by default. When enabled, this adds a settings icon in the top right of the grid, 
+        * which floats above the column header. The menu by default gives access to show/hide columns, 
+        * but can be customized to show additional actions.
+        * @default false
+        */
+        enableGridMenu?: boolean;
```

Comment:
@ashwin027 so it is. It appears from the tutorial that this is a global option, not specific to any specific module for ui-grid.  Therefore, I give my :+1: 

It would still be nice if the ui-grid API docs were updated to include this option (you can do this by going here: https://github.com/angular-ui/ui-grid/edit/master/src/js/core/factories/GridOptions.js)

@ashwin027 , thanks for your contribution!

@vvakame , ready to merge!